src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -182,14 +182,18 @@ class FrontendGenImpl {
   }
 
   void BindOnnxName(const std::string &onnx_name, mlir::Value symbol) {
-    auto input_tensor_legalized_name = legalize_name(onnx_name);
+    std::string result;
+    result = onnx_name;
+    auto input_tensor_legalized_name = result;
```

Comment:
Why the extra lines to do nothing? Won't ```auto input_tensor_legalized_name = onnx_name``` or ```std::string input_tensor_legalized_name = onnx_name``` work (in case a local copy is required)? Similar questions several times below.

Reply:
Sorry for this confusion, the last commit is generated using automatic refactoring tools.  I should've marked this patch as WIP.